src/vcpkg/statusparagraph.cpp

```diff
@@ -9,11 +9,13 @@ namespace vcpkg
 
     void serialize(const StatusParagraph& pgh, std::string& out_str)
     {
+        auto want_literal = to_string_literal(pgh.want);
+        auto state_literal = to_string_literal(pgh.state);
         serialize(pgh.package, out_str);
         out_str.append("Status: ")
```

Comment:
```suggestion
        Strings::append(out_str, "Status: ", want_literal, " ok ", state_literal, "\n");
```

Reply:
I don't want to do this because in #1514 I replaced all of this with the normal 'the way we write paragraph fields' function.

Reply:
https://github.com/microsoft/vcpkg-tool/pull/1514/files#diff-9a667bad62ef21ee1136213bd783a2268ee9f60b9311e8fa7c005cc0090b21bcR80

```c++
append_paragraph_field(ParagraphIdStatus, pgh.status.to_string(), out);
```